code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -109,6 +109,14 @@
 /datum/eldritch_knowledge/proc/on_eldritch_blade(atom/target,mob/user,proximity_flag,click_parameters)
 	return
 
+/**
+  * Sickly blade distant act
+  *
+  * Same as [/datum/eldritch_knowledge/proc/on_eldritch_blade] but works on distant targets
+  */
+/datum/eldritch_knowledge/proc/on_distant_eldritch_blade(atom/target,mob/user,proximity_flag,click_parameters)
```

Comment:
I'll be honest, without context and going to check further code I have no idea what this does, and neither the proc name nor the description offer any insight.